generic/sagemaker/endpoint/variables.tf

```
@@ -39,6 +39,12 @@ variable "endpoint_deployment_config" {
   default     = {}
 }
 
+variable "multi_model" {
```

to be constant with the other variables 
```suggestion
variable "model_multi_model_mode" {
```